src/test/java/edu/hm/hafner/util/StringContainsUtilsTest.java

```diff
@@ -30,5 +38,14 @@ void shouldSearchStrings() {
         assertThat(containsAnyIgnoreCase("This is a string text.", "this")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "is")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "IS")).isTrue();
+
+        assertThat(containsAllIgnoreCase("This is a string text.", "something")).isFalse();
+
+        assertThat(containsAllIgnoreCase("This is a string text.", "This")).isTrue();
+        assertThat(containsAllIgnoreCase("This is a string text.", "this")).isTrue();
+        assertThat(containsAllIgnoreCase("This is a string text.", "wrong", "is")).isFalse();
+        assertThat(containsAllIgnoreCase("This is a string text.", "true", "IS")).isFalse();
+
```

Comment:
Hier wäre es sinnvoll, tatsächlich mal mehr als einen String zu testen.
```suggestion
assertThat(containsAllIgnoreCase("This is a string text.", "this", "IS")).isTrue();
```